app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+
+        vaccine_info = visitor["vaccine"]
+        if "expiration_date" not in vaccine_info:
+            raise ValueError("Vaccine information is incomplete.")
+
+        exp_day = vaccine_info["expiration_date"]
+        if exp_day < datetime.date.today():

```

Comment:
The comparison `if exp_day < datetime.date.today():` is correct, but ensure that `exp_day` is a `datetime.date` object. If `exp_day` is not a date object, this comparison will raise an error.